lib/render.js

```diff
@@ -136,6 +136,17 @@ function generateSchema(value, tree, properties, ignoreDefault = false) {
       if (value.nullable) {
         properties[tree[i]].nullable = true;
       }
+      if (value.type == 'array') {
+        // The last element of the tree is an array. It is a plain or empty array since if not, the tree would have more elements
+        if (value.value == null || value.value.length === 0) {
+          // When it is an empty array
+          properties[tree[i]].items = {
+            type: "", //TODO: how do we know the type in empty arrays?
+          };
+        } else {
+          properties[tree[i]].items = { type: (typeof value.value[0]) };
```

Comment:
Probably this is an edge case, but take into account that there is no 100% overlap between the JSON schema types and the possible values of the `typeof` function.

JSON Schema types ([reference](https://json-schema.org/understanding-json-schema/reference/type.html))
````
string
number
integer
object
array
boolean
null
````

`typeof` returned types ([reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof)):

````
undefined
object
boolean
number
bigint
string
symbol
function
object
````

For instance, beware of the `null` value. It would yield `object` instead of the string `"null"` you are expecting.

Reply:
In principle the only values that the `value` file will contain at that point would be valid for the schema.

For example, it cannot be `null` at that point of the code